mindmeld/components/schemas.py

```diff
@@ -397,10 +398,17 @@ class FormEntitySchema(Schema):
     value = fields.Dict(allow_none=True)
     default_eval = fields.Boolean(default=True)
     hints = fields.List(fields.String(), allow_none=True)
-    custom_eval = fields.String(allow_none=True)
+    custom_eval = fields.Method("serialize_custom_eval",
+                                default=None)
+    def serialize_custom_eval(self, form):  # pylint: disable=no-self-use
+        if form.custom_eval:
+            f = open("custom_eval", "wb")
```

Comment:
Better to do this with a context manager to ensure the file is closed without having to do it explicitly:
```python
with open('custom_eval', 'wb') as f:
    pickle.dump(form.custom_eval, f)
```